spec/teacher_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'spec_helper'
+
+describe Teacher do
+  teacher = Teacher.new(28, 28, 'History', name: 'pame')
+
+  it 'should have a name' do
+    expect(teacher.name).to eql('pame')
+  end
+  it 'should have an age' do
+    expect(teacher.age).to eql(28)
+  end
+
+  it 'should have a specialization in History' do
+    expect(teacher.specialization).to eql('History')
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 🥳  Just one thing miss, Kindly make sure the `can_use_services?` class method has (at least) an 'it' block 👇🏻 check your teacher,rb file

```ruby
  # Method can_use_services?
  def can_use_services?
    true
  end
```